controls/views.py

```diff
@@ -633,6 +633,20 @@ def control_exists_in_catalog(self, catalog_key, control_id):
             control = catalog.get_control_by_id(control_id)
             return True if control is not None else False
 
+def add_selected_components(system, import_record):
```

Comment:
I needed this exact function in #1336 and wrote almost the exact same thing. If this gets merged first, I'll rebase #1336 to use this function. Thanks!